client/state/data-getters/index.js

```diff
@@ -220,7 +224,10 @@ export const requestSitePost = ( siteId, postId, postType ) => {
 			},
 			{}
 		),
-		{ fromApi: () => post => [ [ `gutenberg-site-${ siteId }-post-${ postId }`, post ] ] }
+		{
+			freshness,
```

Comment:
I guess passing `undefined` here will not cause issues?

Reply:
yup defaults to Infinity